### PR TITLE
docs: icon and branding update

### PR DESCRIPTION
**Pull request by tianzhou**

### CODE REVIEW

**Review by d-bytebase — APPROVED** [review]

**Review by LiuJi-Jim — APPROVED** [review]
LGTM

File: frontend/src/components/TheHeader.vue [hunk 1]
```diff
@@ -2,31 +2,36 @@
...
+          href="https://github.com/bytebase/dbcost"
+          data-color-scheme="no-preference: light; light: light; dark: dark;"
+          data-show-count="false"
+          aria-label="Star bytebase/dbcost on GitHub"
+          >Star</a
+        >
+      </div>
 
-      <div class="flex items-center -mt-1.5">
+      <div class="-mt-0.5 flex items-center">
```
Comment (zhouzilong2020) [inline frontend/src/components/TheHeader.vue:26]:
```suggestion
      <div class="mt-0.5 flex items-center">
```

Comment (zhouzilong2020) [reply frontend/src/components/TheHeader.vue:26]: Is this a typo?

Comment (tianzhou) [reply frontend/src/components/TheHeader.vue:26]: This is intentional. Otherwise the icon doesn't align with the github button

**Pull request merged**